assets/app/views/project.html

```diff
@@ -41,7 +41,7 @@ <h1 title="{{project | displayName}}">Project {{project | displayName}}</h1>
 
             <div class="component-block">
               <div class="component">
-                <div class="component-label">
+                <div ng-attr-title="{{service | serviceImplicitDNSName}}" class="component-label">
```

Comment:
My understand is that from within pods you can reference services by their short names like `database`, so the longer name isn't too important.